rdr_service/dao/bq_workbench_dao.py

```diff
@@ -72,8 +72,9 @@ def make_bqrecord(self, src_pk_id, convert_to_enum=False):
             data['income_level'] = str(WorkbenchWorkspaceIncomeLevel(row.income_level))
             data['income_level_id'] = int(WorkbenchWorkspaceIncomeLevel(row.income_level))
 
-            data['access_tier'] = str(WorkbenchWorkspaceAccessTier(row.access_tier))
-            data['access_tier_id'] = int(WorkbenchWorkspaceAccessTier(row.access_tier))
+            if row.access_tier:
```

Comment:
Is there any need to check for the presence of the other `row.*` values above, before trying to cast  as enum values?

Reply:
No, the other enum fields have a default value of zero.